oaknational/Oak-Web-Application
# fix: aria-pressed black/white to true/false

## Description
Bug with `aria-pressed` state appearing as `black`/`white` rather than `true`/`false` in some instances.

## How to test

1. Go to https://deploy-preview-2551--oak-web-application.netlify.thenational.academy/teachers/curriculum/science-secondary-edexcel/units
2. Check all the `aria-pressed` states are `true`/`false` and none are `black`/`white`
